pages/rolodex.js

```diff
@@ -0,0 +1,85 @@
+window.$ = window.jQuery = require('jquery')
+var $ = window.$
```

Comment:
Please use the more idiomatic one-liner
```
var $ = window.$ = window.jQuery = require('jquery')
```
Actually, now I think of it,  `rolodex.js` isn't responsible for ensuring jquery has been properly set up on the window object. That's a job for `main.js`, our entry point.